.spelling

```diff
@@ -415,6 +420,9 @@ git
 GitHub
 GitOps
 GKE
+GKE-IAM
```

Comment:
I expect some of these entries may have crept in from other PRs.

Reply:
yep